content/docs/1.4.0/best-practices.md

```diff
@@ -94,7 +94,7 @@ If you're using `ext4` as the filesystem of the volume, we recommend adding a li
 
 ## Volume Maintenance
 
-We highly recommend using the built-in backup feature of Longhorn.
+Using Longhorn's built-in backup feature is highly recommended. You can save backups to an object store such as S3 or to an NFS server. Saving to an object store is preferable because it generally offers better performance and reliability.  Another advantage is that you do not need to mount and unmount the target, which can complicate failover and upgrades.
```

Comment:
The statement of "better performance" is probably incorrect. I would suggest removing it and only keeping "better reliability".